src/third_party/CMakeLists.txt

```diff
@@ -6,5 +6,9 @@ set(CLIPPER2_EXAMPLES OFF)
 set(CLIPPER2_TESTS OFF)
 set(CLIPPER2_USINGZ "OFF" CACHE STRING "Preempt cache default of USINGZ (we only use 2d)")
 add_definitions(-D_HAS_EXCEPTIONS=0) # disable exceptions for STL
-
 add_subdirectory(clipper2/CPP)
+
+file(GLOB_RECURSE QH_SOURCE_FILES CONFIGURE_DEPENDS quickhull/*.cpp)
```

Comment:
I seem to recall it's better practice to list out the files specifically than to use GLOB.

Reply:
Sure, I was just mimicing what was done in the manifold CMakeLists.